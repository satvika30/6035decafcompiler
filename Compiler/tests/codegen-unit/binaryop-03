class Program
{
    void main()
    {
   		boolean a, b, c; 
   		a = false;
   		b = true;
   		
   		c = a || b;
   		
   		callout("printf", "false || true : %d\n", c);
   		
   		c = a && b ;
   		
   		callout("printf", "false && true : %d\n", c);  		

    }
}
