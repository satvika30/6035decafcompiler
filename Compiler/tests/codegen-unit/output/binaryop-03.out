false || true : 1
false && true : 0
